tests/end-to-end/external-solution-integration/internal/scenario/event_mesh/event_mesh.go

```diff
@@ -67,16 +69,15 @@ func (s *Scenario) Steps(config *rest.Config) ([]step.Step, error) {
 				s.applicationGroup, appOperatorClientset.ApplicationconnectorV1alpha1().Applications(),
 				httpSourceClientset.HTTPSources(kymaIntegrationNamespace)),
 		),
-		step.Parallel(
-			testsuite.NewCreateMapping(s.testID, appBrokerClientset.ApplicationconnectorV1alpha1().ApplicationMappings(s.testID)),
-			testsuite.NewDeployFakeLambda(s.testID, helpers.LambdaPayload, helpers.LambdaPort,
-				coreClientset.AppsV1().Deployments(s.testID),
-				coreClientset.CoreV1().Services(s.testID),
-				coreClientset.CoreV1().Pods(s.testID),
-				true),
-			testsuite.NewStartTestServer(testService),
-			testsuite.NewConnectApplication(connector, state, s.applicationTenant, s.applicationGroup),
-		),
+		testsuite.NewCreateMapping(s.testID, appBrokerClientset.ApplicationconnectorV1alpha1().ApplicationMappings(s.testID)),
+		testsuite.NewDeployFakeLambda(s.testID, helpers.LambdaPayload, helpers.LambdaPort,
+			coreClientset.AppsV1().Deployments(s.testID),
+			coreClientset.CoreV1().Services(s.testID),
+			coreClientset.CoreV1().Pods(s.testID),
+			true),
+		testsuite.NewStartTestServer(testService),
+		testsuite.NewSleep(sleep(s)),
```

Comment:
For what condition to be fulfilled are we sleeping? Instead of just waiting a specified time, can we wait for a specific condition to happen? Something get's ready?

Reply:
This is why I've want to have the second step to solve all these issue, see: https://app.zenhub.com/workspaces/kyma---all-repositories-5b6d5985084045741e744dea/issues/kyma-project/kyma/8409